lms/djangoapps/grades/management/commands/compute_grades.py

```diff
@@ -84,10 +91,11 @@ def enqueue_compute_grades_for_course_tasks(self, course_key, options):
             # created, the most recent enrollments may not get processed.
             # This is an acceptable limitation for our known use cases.
             task_options = {'routing_key': options['routing_key']} if options.get('routing_key') else {}
+            batch_size = ComputeGradesSetting.batch_size if options.get('from_settings') else options['batch_size']
```

Comment:
I may just be ignorant of how ConfigModels work, but this looks like it's just getting the column (on the model class), not the value (on a specific instance).  Also, you should add some tests to the command tests (`lms/djangoapps/grades/management/commands/tests/test_compute_grades.py`) that show the right things happen when the config model is used.